index.js

```diff
@@ -56,6 +57,15 @@ export const run = async () => {
     }
 
     const re = RegExp(regex, regexFlags);
+
+    if (appendRegexToReplacement === "true") {
+        const newOutput = output.match(re);
```

Comment:
suggestion: it seems like it would be more flexible to use a different regex for this rather than being restricted to the same one used for PR body text replacement. Thoughts?

Reply:
I completely agree, that was my initial thought. This does however require another regexFlags as well and I wanted my change to have as minimal impact as possible. Right now, it is possible to replace a snippet with the same regex, like performe in the tests. 

I however do not mind changing it to 2 separate Regex for optimal flexibility.

Reply:
I would go splitting this into two then. Maybe we can use the second regex being present as the truthy check rather than needing `appendRegexToReplacement` to minimize the number of required action inputs.

Since this is new non-breaking functionality I'd also change the version change from a patch to minor bump.